drivers/hopwa_caper/app/controllers/hopwa_caper/cells_controller.rb

```diff
@@ -13,11 +15,11 @@ def show
       @cell = @report.valid_cell_name(params[:id])
       @table = @report.valid_table_name(params[:table])
       @enrollments = @report.hopwa_caper_enrollments.
-        preload(:enrollment).
+        preload(:enrollment, { enrollment: :project }).
         joins(hud_reports_universe_members: { report_cell: :report_instance }).
         merge(::HudReports::ReportCell.for_table(@table).for_cell(@cell))
       @services = @report.hopwa_caper_services.
-        preload(enrollment: :enrollment).
+        preload(enrollment: [:enrollment, { enrollment: :project }]).
```

Comment:
Same here, I _think_ this is the same as:
```suggestion
        preload(enrollment: { enrollment: :project }).
```

Reply:
added [here](https://github.com/greenriver/hmis-warehouse/pull/5203/commits/b1e0355cd6a6b7c93f77f0c4026cbcbd615f9ddc)